src/Slider/Slider.tsx

```diff
@@ -306,7 +306,7 @@ export const Slider: RneFunctionComponent<SliderProps> = ({
     if (!allowTouchTrack && !TRACK_STYLE) {
       return thumbHitTest(e);
     }
-    if (!trackStyle) {
+    if (!trackStyle && allowTouchTrack) {
```

Comment:
Shouldn't this be a ||

@saif-malik-01 
@flyingcircle Does that work if you try with ||??

Reply:
Nope , it will not. It actually make no sense  to the statement as no matter what the value of prop is, this statement is going to be always evaluate as true .

Reply:
This doesn't logically change anything. Combined with the if statement above, adding this doesn't do anything.